frontend/stylesheets/trace_view/_tracing_view.less

```diff
@@ -42,10 +39,15 @@
   &.TDView {
     border-color: white;
   }
-
-  &:hover {
-    opacity: 1;
-  }
+}
+.inputcatcher {
+  z-index: 20;
+  box-sizing: border-box;
+  -moz-box-sizing: border-box;
```

Comment:
This should no longer be needed since the element doesn't have a border. I'm not sure whether it should be moved to `inputcatcher_border` instead :see_no_evil:

Reply:
You are right. I think, it can be removed altogether, because no code is interested in measuring (or sizing) the border-only container.